ddpui/api/transform_api.py

```
@@ -1412,3 +1411,34 @@ def get_canvas_node(request, node_uuid: str):
+    try:
+        warehouse_obj = OrgWarehouse.objects.get(org=org)
+    except OrgWarehouse.DoesNotExist:
+        raise HttpError(400, "Organization does not have a warehouse configured")
+
+    try:
+        # Parse manifest to canvas
+        stats = parse_dbt_manifest_to_canvas(org, org.dbt, warehouse_obj)
+
+        logger.info(f"Manifest sync completed for org {org.name}. Stats: {stats}")
+
+        return {"success": True, "message": "Manifest sync completed successfully", "stats": stats}
```

Ruff (0.14.8)

Consider moving this statement to an `else` block

(TRY300)